packages/chat-stateful-client/review/chat-stateful-client.api.md

```diff
@@ -70,6 +70,9 @@ export type ChatThreadProperties = {
 // @public
 export const createStatefulChatClient: (args: StatefulChatClientArgs, options?: StatefulChatClientOptions | undefined) => StatefulChatClient;
 
+// @internal
+export const _createStatefulChatClientWithDeps: (chatClient: ChatClient, args: StatefulChatClientArgs, options?: StatefulChatClientOptions | undefined) => StatefulChatClient;
```

Comment:
(name police)
if we're passing in a reference to the chatClient are we still creating it in this function?

Reply:
We _are_ creating the *stateful* chat client :)